source/getting-started/core-concepts.md

```diff
@@ -1,82 +1,59 @@
-powerful: the URL.
-
-### Concepts
-
-#### Templates
-
-A **template**, written in the Handlebars templating language, describes
-the user interface of your application. Each template is backed by a
-model, and the template automatically updates itself if the model changes.
-
-In addition to plain HTML, templates can contain:
-
-* **Expressions**, like `{{firstName}}`, which take information from
-  the template's model and put it into HTML.
-* **Outlets**, which are placeholders for other templates. As users
-  move around your app, different templates can be plugged into the
-  outlet by the router. You can put outlets into your template using the
-  `{{outlet}}` helper.
-* **Components**, custom HTML elements that you can use to clean up
-  repetitive templates or create reusable controls.
-
-#### Router
-
-The **router** translates a URL into a series of nested templates, each
-backed by a model. As the templates or models being shown to the user
-change, Ember automatically keeps the URL in the browser's address bar
-up-to-date.
-
-This means that, at any point, users are able to share the URL of your
-app. When someone clicks the link, they reliably see the same content as
-the original user.
-
-#### Components
-
-A **component** is a custom HTML tag whose behavior you implement using
-JavaScript and whose appearance you describe using Handlebars templates.
-They allow you to create reusable controls that can simplify your
-application's templates.
-
-#### Models
-
-A **model** is an object that stores _persistent state_. Templates are
-responsible for displaying the model to the user by turning it into
-HTML. In many applications, models are loaded via an HTTP JSON API,
-although Ember is agnostic to the backend that you choose.
-
-
-#### Routes
-
-A **route** is an object that tells the template which model it should
-display.
-
-These are the core concepts you'll need to understand as you develop
-your Ember.js app. They are designed to scale up in complexity, so that
-adding new functionality doesn't force you to go back and refactor major
-parts of your app.
-
-Now that you understand the roles of these objects, you're equipped to
-dive deep into Ember.js and learn the details of how each of these
-individual pieces work.
+## Templates
+
+Templates, written in the Handlebars language, describe the user interface of
+your application. In addition to plain HTML, templates can contain expressions,
+like `{{title}}` or `{{author}}`, which take information from a component and
+put it into HTML. They can also contain helpers, like `{{if isAdmin}}30 people
+have viewed your blog today.{{/if}}.` Finally, they can contain components, such
+as if a template listing out all the blog posts in an app rendered a component
+for each post.
+
+## Components
+
+Components are the primary way user interfaces are organized in Ember. They
+consist of two parts: a template, and JavaScript that defines the component's
```

Comment:
That sounds good to me. I wish we could just call it a "component controller", but that would be confusing, I guess. There really should be a name for that "source file written in JavaScript".